commands/azureCommands/acr-logs-utils/tableViewManager.ts

```diff
@@ -0,0 +1,270 @@
+
+import { ImageDescriptor, Run } from "azure-arm-containerregistry/lib/models";
+import * as clipboardy from 'clipboardy'
+import * as path from 'path';
+import * as vscode from "vscode";
+import { accessLog } from './logFileManager';
+import { Filter, LogData } from './tableDataManager'
+export class LogTableWebview {
+    private logData: LogData;
+    private panel: vscode.WebviewPanel;
+
+    constructor(webviewName: string, logData: LogData) {
+        this.logData = logData;
+        this.panel = vscode.window.createWebviewPanel('log Viewer', webviewName, vscode.ViewColumn.One, { enableScripts: true, retainContextWhenHidden: true });
+
+        //Get path to resource on disk
+        let extensionPath = vscode.extensions.getExtension("PeterJausovec.vscode-docker").extensionPath;
```

Comment:
Pass in extension context for this rather than using getExtension with hard-coded string.

Reply:
I've been trying to do what you're asking, but not really sure how, and I can't figure it out. Could you give me an example of how to do this?

Reply:
Sorry, just use this: ext.context.extensionPath